include/swift/ABI/Metadata.h

```diff
@@ -79,6 +79,16 @@ struct RuntimeTarget<4> {
   using StoredSize = uint32_t;
   using StoredPointerDifference = int32_t;
   static constexpr size_t PointerSize = 4;
+
+#if SWIFT_OBJC_INTEROP
```

Comment:
You'll want to pull this out of `RuntimeTarget` and use `WithObjCInterop` or `NoObjCInterop` as appropriate instead. The idea is that the target's ObjC interop setting may not match what we're built with. See https://github.com/apple/swift/pull/40927/files for this recent change. /cc @adrian-prantl

Reply:
Yeah this is important. LLDB will instantiate various versions of this to support remote debugging where LLDB is built with SWIFT_OBJC_INTEROP but the remote target may not be.

Reply:
Great catch - I'd already adjusted all the use-sites to use `WithObjCInterop` and `NoObjCInterop` and this was leftover from a previous iteration. Removed, thanks!